City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-5430: Playgroup clubs

# Daycare options filter value change [UHF-5430](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5430)
Use "Playgroup clubs" instead of "Open family activities" with the Daycare options filter.

## How to install
* Make sure your instance is up and running.
* Get the changes:
  * `git pull origin UHF-5430-playgroup-clubs`
  * `composer require drupal/helfi_platform_config:dev-UHF-5430-change-texts`
* Run `make` and:
  * `drush cim`
  * `drush locale-import fi /app/public/modules/contrib/helfi_platform_config/translations/new/fi.po`
  * `drush locale-import sv /app/public/modules/contrib/helfi_platform_config/translations/new/sv.po`
  * `drush cr`
  * `drush migrate:import tpr_ontology_word_details`

## How to test
* Check that Daycare search has the "Playgroup clubs" as one of the options value.
* Check that selecting this option there are some daycares, e.g. Daycare Kulosaari.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/310
